r124/instream.csh

```diff
@@ -30,7 +31,7 @@ setenv subtest_list_common     ""
 setenv subtest_list_non_replic ""
 setenv subtest_list_non_replic "$subtest_list_non_replic readonly ydb275socketpass ydb280socketwait jnlunxpcterr ydb297"
 setenv subtest_list_replic     ""
-setenv subtest_list_replic     "$subtest_list_replic ydb282srcsrvrerr ydb293"
+setenv subtest_list_replic     "$subtest_list_replic ydb282srcsrvrerr ydb293 ydb312_gtm8182c"
```

Comment:
Can you also resolve conflicts with latest upstream master while addressing the other comments.